arm-hdinsight/2015-03-01-preview/swagger/cluster.json

```diff
@@ -96,62 +96,12 @@
                 },
                 "x-ms-long-running-operation": true
             },
-            "patch": {

```

Comment:
We don't actually surface any such API currently. Not sure why it was here in the first place.